src/http/server.cr

```diff
@@ -258,6 +258,11 @@ class HTTP::Server
     def bind_tls(address : Socket::IPAddress, context : OpenSSL::SSL::Context::Server) : Socket::IPAddress
       bind_tls(address.address, address.port, context)
     end
+
+    # DEPRECATED: Use `#bind_tls`.
```

Comment:
Please add an explicit `TODO: remove in 0.27.0` or similar

we should probably eventually settle on a `TODO(0.27.0)` style with a `grep -R "TODO($(crystal -v))" src spec` to fail the build.

Reply:
done.